source/fundamentals/encrypt-fields/qe.txt

```diff
@@ -0,0 +1,80 @@
+.. _node-qe:
+
+==============================
+Configure Queryable Encryption
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to configure the {+driver-short+} for {+qe+}.
+
+For thorough coverage of the core concepts, use-cases, examples, and
+reference, visit the main {+qe+} :ref:`documentation site <qe-manual-feature-qe>`.
+
+Introduction
+------------
+
+{+qe+} allows you to encrypt specific data fields within a document before
+sending the data to the server, and to query on encrypted fields. {+qe+}
+is useful for situations in which applications must guarantee that
+unauthorized parties, including server administrators, cannot read the
+encrypted data.
+
+
+Configuration
+-------------
+
+To use {+qe+}, you must:
+
+- Configure the driver with specific settings for encryption.
+- Configure and use the driver's encryption package, ``mongodb-client-encryption``.
+
+
+MongoClient Configuration
++++++++++++++++++++++++++
+
+You must pass an ``AutoEncrytionOptions`` object to the ``MongoClient`` constructor
+to enable {+qe+} operations with the {+driver-short+}.
+
+.. include:: /includes/auto-encryption-options.rst
+
+.. example::
+
+   .. literalinclude:: /includes/qe/qe-options.js
+      :start-after: begin qe mongoclient
+      :end-before: end qe mongoclient
+      :language: javascript
+      :dedent:
+
+ClientEncryption Configuration
+++++++++++++++++++++++++++++++
+
+``ClientEncryption`` is a class exposed by ``mongodb-client-encryption`` that provides
+methods for interacting with your {+dek-long+}s. You construct a ``ClientEncryption`` object
+by passing a ``MongoClient`` object to the ``ClientEncryption`` constructor, and
+configuration options.
+
+.. include:: /includes/client-encryption-options.rst
+
+.. example::
+
+   .. literalinclude:: /includes/qe/qe-options.js
+      :start-after: begin client encryption
+      :end-before: end client encryption
+      :language: javascript
+      :dedent:
+
+Further Reading
+---------------
+
+For thorough coverage of the core concepts, use-cases, examples, and
+reference, visit the main {+qe+} :ref:`documentation site <qe-manual-feature-qe>`.
```

Comment:
```suggestion
For coverage of the core concepts, use-cases, examples, and
reference, visit the :ref:`{+qe+} documentation site <qe-manual-feature-qe>`.
```

https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-examples